src/styles/install.css

```diff
@@ -1,10 +1,9 @@
 .sto-install {
   height: 100%;
-  max-width: 32rem;
+  width: 32rem;
```

Comment:
How this is displayed on mobile with screens < 32rem ?

Reply:
Pretty well actually 👌 
![capture d ecran 2018-05-03 a 17 15 04](https://user-images.githubusercontent.com/776764/39585640-9585a8a4-4ef5-11e8-80b0-70203d538ad3.png)